docs/environments/pacman.md

```diff
@@ -0,0 +1,66 @@
+# PacMan Environment
+
+<p align="center">
+        <img src="../env_anim/pacman.gif" width="600"/>
+</p>
+
+We provide here a minimal Jax JIT-able implementation of the game pacman. The game is played in a 2D matrix where a cell is a free space (black), a wall (dark blue), pacman (yellow) or a ghost.
```

Comment:
Could we provide a reference to the game? For instance this:
```suggestion
We provide here a minimal Jax JIT-able implementation of the game [PAC-MAN](https://pacman.com/en/history/). The game is played in a 2D matrix where a cell is a free space (black), a wall (dark blue), pacman (yellow) or a ghost.
```
But please, feel free to use a better reference.